libs/remix-ui/settings/src/lib/remix-ui-settings.tsx

```diff
@@ -0,0 +1,171 @@
+import React, { useState, useReducer, useEffect, useCallback } from 'react' // eslint-disable-line
+import { CopyToClipboard } from '@remix-ui/clipboard' // eslint-disable-line
+
+import { generateContractMetadataText, gitAccessTokenLink, gitAccessTokenText, gitAccessTokenText2, gitAccessTokenTitle, warnText } from './constants'
+
+import './remix-ui-settings.css'
+import { etherumVM, generateContractMetadat, personal, textWrapEventAction, useMatomoAnalytics } from './settingsAction'
+import { initialState, settingReducer } from './settingsReducer'
+import { Toaster } from '@remix-ui/toaster'// eslint-disable-line
+
+/* eslint-disable-next-line */
+export interface RemixUiSettingsProps {
+  config: any,
+  editor: any,
+   _deps: any
+}
+
+export const RemixUiSettings = (props: RemixUiSettingsProps) => {
+  const [, dispatch] = useReducer(settingReducer, initialState)
+  const [tokenValue, setTokenValue] = useState('')
+  const [saveTokenState, setSaveTokenState] = useState(false)
+  const [removeTokenState, setRemoveTokenState] = useState(false)
+  const [themeName, setThemeName] = useState('')
+
+  useEffect(() => {
+    props._deps.themeModule.switchTheme()
+    const token = props.config.get('settings/gist-access-token')
+    if (token === undefined) {
+      props.config.set('settings/generate-contract-metadata', true)
+      dispatch({ type: 'contractMetadata', payload: { name: 'contractMetadata', isChecked: true, textClass: 'text-dart' } })
```

Comment:
Is this a typo? `text-dart`

Reply:
sure, it was  a typo, have corrected that